packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -136,21 +139,23 @@ export const HelpCenterMoreResources = () => {
 						</a>
 					</div>
 				</li>
-				<li className="inline-help__resource-item">
-					<div className="inline-help__resource-cell">
-						<Button
-							variant="link"
-							onClick={ handleWhatsNewClick }
-							className="inline-help__new-releases"
-						>
-							<Icon icon={ <NewReleases /> } size={ 24 } />
-							<span>{ __( "What's New", __i18n_text_domain__ ) }</span>
-							{ showWhatsNewDot && (
-								<Icon className="inline-help__new-releases_dot" icon={ circle } size={ 16 } />
-							) }
-						</Button>
-					</div>
-				</li>
+				{ showWhatsNewItem && (
```

Comment:
I wonder if we can just reuse the `showWhatsNewDot` variable, since it's supposed to only light up when there are new and unread items?

Reply:
This would mean the Whats New button would no longer be available once all active announcements have been seen once, and users would no longer be able to re-open it.

If there are any active items for whats-new, I think we should keep access to the button. Regardless of if they have been marked as seen yet or not.

Reply:
Oh, you're right. I assumed the data.length would be zero once all are viewed, but that's incorrect.